tests/database/test_base_db_api.py

```diff
@@ -80,7 +80,7 @@ def test_database_api_missing_key_retrieval(db):
 
 
 def test_database_api_missing_key_for_deletion(db):
-    db.delete(b'does-not-exist')
```

Comment:
I think the idea here was that `delete()` kind of mirrors `dict.get` which returns `None` instead of raising an error. ie:
https://github.com/ethereum/py-evm/blob/bad0c8e694eb34e14590461d3a9b06a25624d745/eth/db/backends/base.py#L44-L48

So this line was testing that `.delete()`ing a missing key doesn't crash. Admittedly it wasn't documented anywhere and it should maybe be in its own test. :see_no_evil: